src/registry/domain/plugins-initialiser.js

```diff
@@ -98,6 +98,7 @@ module.exports.init = function(pluginsToRegister, callback) {
     plugin.register.register(plugin.options || {}, dependencies, err => {
       const pluginCallback = plugin.callback || _.noop;
       pluginCallback(err);
+      plugin.register.execute.toString = () => plugin.description || '';
       registered[plugin.name] = plugin.register.execute;
```

Comment:
I think it would be cleaner if we could do something like this:
```javascript
registered[plugin.name] = {
  execute:...
  description: ...
}
``` and then use those properties accordingly.

Reply:
but that's not what we pass to the registry

Reply:
we've clarified this on a call. we probably want to overload toString anyway. Comment is added to clarify this bit in the code as it is not obvious